utils/getRandomColor.ts

```diff
@@ -0,0 +1,48 @@
+export type RabbitColor = {
+  eyeColor: string;
+  bodyColor: string;
+};
+
+const palette = [
```

Comment:
```suggestion
const palette: RabbitColor[] = [
```

타입 지정이 있으면 더 좋을 것 같습니다 ;D